components/Layout.js

```diff
@@ -1,6 +1,7 @@
-import {WelcomeCard} from "../components/WelcomeCard";
 import styled from "styled-components";
 import {Icon} from "@iconify/react";
+import Link from "next/link";
+import {WelcomeCard} from "../components/WelcomeCard";
 
 export function Layout() {
   return (
```

Comment:
You don't render children in the Layout component, that's why you don't see the actual page content of your different pages. Change the function signature like this: `export function Layout({children})` and place the `{children}` element at the appropriate place of your layout (I guess between `WelcomeCard` and `NavBar`)

Reply:
Thanks @actyralf ! Ok, the logo and the welcome message are displayed on all pages, and they should only be displayed on the home page